### ISSUE TITLE
WSL 2 shows a KMODE_EXCEPTION_NOT_HANDLED BSOD upon installing a distro.

### ISSUE THREAD

**Issue by catink123** [microsoft/WSL#7514]
### Windows Build Number

Microsoft Windows [Version 10.0.22000.194]

### WSL Version

- [ ] WSL 2
- [ ] WSL 1

### Kernel Version

5.10.60.1

### Distro Version

Ubuntu 20.04

### Repro Steps

Enable Windows Subsystem for Linux feature.
Change to WSL 2 with `wsl --set-default-version 2`.
Install 'Ubuntu' from Microsoft Store.
Launch the Ubuntu application.

### Expected Behavior

Ubuntu installs itself and shows first time setup.

### Actual Behavior

The window shows 'Installing, this may take a few minutes`. Then, after 2-3 seconds the whole system hangs and then a BSOD pops up with error KMODE_EXCEPTION_NOT_HANDLED. The file that it shows is dxgmms2.sys.

**Comment by felpshn** [comment]
I'm having the exact same issue after installed Windows 11. I've tried to find for some solution, but I didn't have any success.. Idk, but seems like it's some kind of incompatibility between the Windows 11 kernel and the graphic drivers currently available - in my case, AMD. Guess we'll have to wait for updates..

**Comment by asvishnyakov** [comment]
Hi guys, did you had a chance to solve issue?

**Comment by catink123** [comment]
Nope, still using Windows 10 with WSL2 as I thought that my processor/motherboard is old enough for everything to break on Windows 11. However, using Windows 11 on a 9 year old unsupported laptop is flawless and even WSL2 works fine. Maybe it's about the difference between Intel and AMD processors.
